jcheck/src/main/java/org/openjdk/skara/jcheck/JCheck.java

```diff
@@ -295,4 +295,16 @@ public static Set<Check> checksFor(ReadOnlyRepository repository, Hash hash) thr
                                 null);
         return jcheck.checksForRange();
     }
+
+    public static Set<Check> checksFor(JCheckConfiguration conf, ReadOnlyRepository repository, Hash hash) throws IOException {
+        var jcheck = new JCheck(repository,
+                                CommitMessageParsers.v1,
+                                repository.range(hash),
+                                Pattern.compile(".*"),
+                                Pattern.compile(".*"),
+                                List.of(),
+                                conf,
+                                null);
+        return jcheck.checksForRange();
+    }
```

Comment:
I think I would like to keep it for the tests since those tests will be exercising slightly different code paths (especially with the most recent version)